src/client/v2/algod/getLedgerStateDeltaForTransactionGroup.ts

```diff
@@ -0,0 +1,15 @@
+import JSONRequest from '../jsonrequest';
+import HTTPClient from '../../client';
+import IntDecoding from '../../../types/intDecoding';
+
+export default class GetLedgerStateDeltaForTransactionGroup extends JSONRequest {
```

Comment:
Is there no generated type for a `StateDelta` object?

Reply:
I forgot this was hand generated and not specified in the OAPI spec -- resolving

Reply:
Right, for now the StateDelta will be under-specified. Until we have something like state commitments that puts ledger objects behind consensus I don't feel comfortable maintaining delta objects and their internals for SDKs.